### PR TITLE
Added SDK examples of AAD advanced queries

### LINKED ISSUE #15711: Information about advanced queries and the fact you need -ConsistencyLevel Eventual and -Countvariable <something> Should be added
It should cover the obtuse error you get about the query and that you will need to do this. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bc826d9c-c7e7-85de-7653-9249f7fade9f
* Version Independent ID: 11df7fb5-969b-98d4-0406-aa16f4cda0a4
* Content: [Navigating the Microsoft Graph PowerShell SDK - Microsoft Graph](https://docs.microsoft.com/en-us/graph/powershell/navigating)
* Content Source: [concepts/powershell/navigating.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/concepts/powershell/navigating.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @jasonjoh
* Microsoft Alias: **MSGraphDocsVteam**

### PR DESCRIPTION
**Pull request by jasonjoh**
Added Note block to various SDK topics directing users to this topic if they get unsupported query errors.

Fixes #15711

### CODE REVIEW

**Review by maisarissi — APPROVED** [review]

File: includes/aad-advanced-queries-note.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+author: "jasonjoh"
+ms.localizationpriority: high
+ms.topic: include
+---
+
+<!-- markdownlint-disable MD041-->
+
+> [!NOTE]
+> Some requests for Azure Active Directory resources require the use of advanced query capabilities. If you get a response indicating a bad request, unsupported query, or a response that includes unexpected results, including the `$count` query parameter and `ConsistencyLevel` header may allow the request to succeed. For details and examples, see [Advanced query capabilities on Azure AD directory objects](../concepts/aad-advanced-queries.md).
```
Comment (FaithOmbongi) [inline includes/aad-advanced-queries-note.md:10]: I think `$count` and `consistencyLevel ` are translated to `-Count` and `-ConsistencyVariable` in Microsoft Graph PowerShell. If this is true, I also suggest that you change the file name to indicate that this include is for PowerShell only.
```suggestion
> Some requests for Azure Active Directory resources require the use of advanced query capabilities. If you get a response indicating a bad request, unsupported query, or a response that includes unexpected results, including the `-Count` query parameter and `-ConsistencyLevel` header may allow the request to succeed. For details and examples, see [Advanced query capabilities on Azure AD directory objects](../concepts/aad-advanced-queries.md).
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @jasonjoh for adding the snippets to the conceptual examples. 

A note that the syntax for the advanced query parameters in Microsoft Graph PowerShell is a bit different: `-Count` and `-ConsistencyLevel`. @msewaweru, kindly confirm.

**Comment by msewaweru** [comment]
> Thank you @jasonjoh for adding the snippets to the conceptual examples.
> 
> A note that the syntax for the advanced query parameters in Microsoft Graph PowerShell is a bit different: `-Count` and `-ConsistencyLevel`. @msewaweru, kindly confirm.

You are right @FaithOmbongi. The parameters should be indicated as **-Count** and **-ConsistencyLevel**.

File: includes/aad-advanced-queries-note.md (hunk 1 above)
Comment (jasonjoh) [reply includes/aad-advanced-queries-note.md:10]: No, this isn't for PowerShell only. It's for all the SDKs. For that reason, I used the actual parameter and header names. The PS example in aad-advanced-queries.md uses the proper parameters.

**Pull request merged**